biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -49,7 +49,7 @@ export class AppController {
    * @param data The payload sent for the created simulation run message
    */
   @MessagePattern(DispatchMessage.created)
-  async uploadFile(data: DispatchCreatedPayload): Promise<createdResponse> {
+  async uploadFile(data: any): Promise<createdResponse> {
```

Comment:
Why is the typing removed here?

Reply:
The same time should be added to the api end to send the payload

Reply:
I wasn't able to get filename when typing was applied. When typing was removed I was getting the filename.
Maybe something was missing in type definition, couldn't exactly figure out why.

Reply:
Resolved. Was due to a typo in the filename previously